administrator/language/de-DE/de-DE.com_privacy.ini

```
@@ -35,11 +35,11 @@ COM_PRIVACY_CORE_CAPABILITY_COMMUNICATION_WITH_JOOMLA_ORG="Wenn eine Netzwerkver
 ; The placeholder for this key is the configured log path for the site.
 COM_PRIVACY_CORE_CAPABILITY_LOGGING_IP_ADDRESS="Joomlas Logging-System zeichnet die IP-Adresse des Besuchers auf und schreibt diesen Eintrag in eine Protokoll-Datei. Diese Protokolldateien werden verwendet, um verschiedene Aktivitäten auf einer Joomla Seite aufzuzeichnen, einschließlich Informationen zu wichtigen Updates, ungültigen Anmeldeversuchen, nicht behandelten Fehlern und Entwicklungsinformationen wie der Verwendung veralteter APIs. Das Format dieser Protokolldateien kann von jeder Erweiterung angepasst werden, die einen Logger verwendet. Die Protokolldateien sollten regelmäßig herunterladen und überprüft werden. Die Protokolldateien sind unter `%s` zu finden."
 COM_PRIVACY_CORE_CAPABILITY_SESSION_IP_ADDRESS_AND_COOKIE="Alle Anfragen an eine Joomla-Website starten eine Sitzung, in der die IP-Adresse in den Sitzungsdaten gespeichert wird und ein Sitzungscookie im Browser des Benutzers erstellt wird. Die IP-Adresse wird als Sicherheitsmaßnahme zum Schutz vor möglichen Session-Hijacking-Angriffen verwendet. Diese Informationen werden gelöscht, sobald die Sitzung abgelaufen und die Daten gelöscht wurden. Der Name des Session-Cookies basiert auf einem zufällig generierten Hash und hat daher keinen konstanten Bezeichner. Der Sitzungscookie wird gelöscht, sobald die Sitzung abgelaufen ist oder der Benutzer den Browser verlassen hat."
-COM_PRIVACY_DASHBOARD_BADGE_ACTIVE_REQUESTS_0="<span class=\"badge badge-info\">%d</span> aktive Anfragen"
-COM_PRIVACY_DASHBOARD_BADGE_ACTIVE_REQUESTS_1="<span class=\"badge badge-warning\">%d</span> aktive Anfragen"
+COM_PRIVACY_DASHBOARD_BADGE_ACTIVE_REQUESTS_0="<span class=\"badge badge-info\">keine</span> aktive Anfrage"
+COM_PRIVACY_DASHBOARD_BADGE_ACTIVE_REQUESTS_1="<span class=\"badge badge-warning\">eine</span> aktive Anfrage"
 COM_PRIVACY_DASHBOARD_BADGE_ACTIVE_REQUESTS_MORE="<span class=\"badge badge-warning\">%d</span> aktive Anfragen"
-COM_PRIVACY_DASHBOARD_BADGE_TOTAL_REQUESTS_0="<span class=\"badge badge-info\">%d</span> Gesamtanfragen"
-COM_PRIVACY_DASHBOARD_BADGE_TOTAL_REQUESTS_1="<span class=\"badge badge-info\">%d</span> Gesamtanfragen"
+COM_PRIVACY_DASHBOARD_BADGE_TOTAL_REQUESTS_0="<span class=\"badge badge-info\">keine</span> Gesamtanfragen"
```

Keine müsste dann groß sein oder?


```suggestion
COM_PRIVACY_DASHBOARD_BADGE_TOTAL_REQUESTS_0="<span class=\"badge badge-info\">Keine</span> Gesamtanfragen"
```